pylintrc

```diff
@@ -1,3 +1,4 @@
 [BASIC]
 # Allow django's urlpatterns, and our log preference
 const-rgx = (([A-Z_][A-Z0-9_]*)|(__.*__)|log|urlpatterns)$
+ignore=migrations
```

Comment:
so this kind of sucks, but this appears to be a limitation of pylint, if you run `pylint learningobjects/migrations/0001_initial.py --ignore=migrations` or `pylint learningobjects/migrations/0001_initial.py --ignore=0001_initial.py` then it still checks the file.  I think we just need to add `#pylint: skip-file` unfortunately.  The pep8 rule will work, and I'll file an issue to see if we can't fix it either in pylint or at least pytest-pylint.